islandora.module

```diff
@@ -970,7 +970,7 @@ function islandora_islandora_object_purged($pid) {
 function islandora_islandora_datastream_purged(AbstractObject $object, $dsid) {
   $cache_meta = CacheableMetadata::createFromObject($object)
     ->addCacheTags([
-      "{$object->id}/{$dsid}",
+      "islandora_object:{$object->id}/{$dsid}",
```

Comment:
Actually, looking at [the implementation](https://github.com/discoverygarden/islandora/blob/bc4d12b12625d6cfba691d60b488a2a377828b84/src/Tuque/IslandoraFedoraDatastream.php#L33-L44), looks like it should be:
```suggestion
      "islandora_object:{$object->id}:{$dsid}",
```
or possibly, could make use of the `$object`:
```suggestion
      $object->drupalCacheTag() . ":{$dsid}",
```

... Alternatively, we could introduce a static method on `IslandoraFedoraDatastream` analogous to [`IslandoraFedoraObject::getDrupalCacheObjectTag()`](https://github.com/discoverygarden/islandora/blob/bc4d12b12625d6cfba691d60b488a2a377828b84/src/Tuque/IslandoraFedoraObject.php#L48-L59).